### PR TITLE
Documentation, isascii() check, news

### PR DESCRIPTION
**Pull request by kbarbary**
I ported the rest of the API docs to docstrings and removed mkdocs, so hopefully the travis doc build will succeed this time. :crossed_fingers:

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -5,13 +5,15 @@ makedocs(
     format = :html,
     sitename = "FITSIO",
     pages    = Any[
-        "Introduction"   => "index.md",
+        "Introduction" => "index.md",
+        "API Reference" => "api.md"
```
Comment (mweastwood) [inline docs/make.jl]: Looks like you forgot to add `api.md`?

Comment (kbarbary) [reply docs/make.jl]: Thanks! I certainly did.

File: docs/src/api.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+write(::FITS, ::Dict{String})
+write(::FITS, ::Vector{String}, ::Vector)
+read(::TableHDU, ::String)
+```
+
+## Miscellaneous
+
+```@docs
+FITSIO.libcfitsio_version
+```
```
Comment (giordano) [inline docs/src/api.md]: I don't know what's your attitude towards newline at the end of a file, anyway here it's missing.

File: src/table.jl [hunk 3]
```diff
@@ -341,6 +341,16 @@ function write_internal(f::FITS, colnames::Vector{String},
...
 
+
+"""
+    write(f::FITS, colnames, coldata; hdutype=TableHDU, name=nothing, ver=nothing, header=nothing, units=nothing, varcols=nothing)
+
+Same as `write(f::FITS, data::Dict; ...)` but providing column names
+and column data as a separate arrays. This is useful for specifying
+the order of the columns. Column names must be `Array{ASCIIString}`
+and column data must be an array of arrays.
+
```
Comment (giordano) [inline src/table.jl]: Extra blank line?

File: src/table.jl [hunk 4]
```diff
@@ -351,6 +361,45 @@ function write(f::FITS, colnames::Vector{String}, coldata::Vector;
...
+    when the rows of a column vary greatly in length, as the column
+    data is all written to a contiguous heap area at the end of the
+    table. Only column data of type `Vector{ASCIIString}` or types
+    such as `Vector{Vector{UInt8}}` can be written as variable
+    length columns. In the second case, ensure that the column data
+    type is a *leaf type*. That is, the type cannot be
+    `Vector{Vector{T}}`, which would be an array of arrays having
+    potentially non-uniform element types (which would not be writable
+    as a FITS table column).
+
```
Comment (giordano) [inline src/table.jl]: Probably also this blank line may be removed.

**Comment by kbarbary** [comment]
Yay! http://juliaastro.github.io/FITSIO.jl/latest/

**Pull request merged**